### PR TITLE
Add 'Time Rift' as trackable progress

### PR DESCRIPTION
**Pull request by Kralizek**
Adds quest 77836 (Time Rift) as stand-alone line in the progress view.

Not sure if more than this is needed.

### CODE REVIEW

File: SavedInstances/Modules/Progress.lua [hunk 1]
```diff
@@ -688,6 +688,19 @@ local presets = {
     persists = false,
     fullObjective = false,
   },
+  -- Time Rift
+  ['df-time-rift'] = {
+    type = 'any',
```
Comment (LiangYuxuan) [inline SavedInstances/Modules/Progress.lua:693]: use 'single' is better

**Review by LiangYuxuan — CHANGES_REQUESTED** [review]

File: SavedInstances/Modules/Progress.lua (hunk 1 above)
Comment (Kralizek) [reply SavedInstances/Modules/Progress.lua:693]: ok

**Review by Kralizek — COMMENTED** [review]
Switched quest type to 'single'

**Comment by LiangYuxuan** [comment]
`questID = 77836,` number but not table.

To find quest id you can try https://www.curseforge.com/wow/addons/questschanged . Super useful for finding hidden quests.

**Comment by Kralizek** [comment]
Not sure I understand what you mean by

> number, not table

**Comment by LiangYuxuan** [comment]
You need to provide `questID` by number but not table for 'single' entries. So you need to change it to `questID = 77836,`. Also for #781 .

**Comment by Kralizek** [comment]
ah ok! :)

**Review by LiangYuxuan — APPROVED** [review]

**Pull request merged**